### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sind14**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list[Customer],
+            cleaning_staff: str) -> None:
+        print(f'\"{movie_name}\" started in hall number {self.number}.')
+        for customer in customers:
+            if isinstance(customer, Customer):
```
Comment (ArsenPidhoretskyi) [inline app/cinema/hall.py:16]: It always accepts customer
Check your annotations
And remove this if-else statement

File: app/cinema/hall.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list[Customer],
+            cleaning_staff: str) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/cinema/hall.py:13]: > movie_session, that takes movie_name, customers - list of a customers (Customer instances), cleaning_staff - cleaner (Cleaner instance).
cleaning_staff is not str

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list[Customer],
+        hall_number: int,
+        cleaner: Cleaner(Customer),
```
Comment (ArsenPidhoretskyi) [inline app/main.py:10]: > cinema_visit that takes movie_name, customers - a list of customers, elements are dicts with 'name' and desired 'food' of a customer, cleaning_staff - name of the cleaner, that will clean the hall after movie session

Wrong type annotations
customers -> list of dicts
cleaning_staff -> str

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,14 @@
...
+        movie: str) -> None:
+    for customer in customers:
+        CinemaBar.sell_product(customer["food"], customer["name"])
```
Comment (ArsenPidhoretskyi) [inline app/main.py:13]: You must create customers objects inside cinema_visit

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Task is not completed

**Review by Bohdan-Salamakha — APPROVED** [review]

**Pull request closed without merging**